deps/curl-7.88.1/lib/vtls/sf_ocsp.c

```diff
@@ -638,9 +687,10 @@ static OCSP_RESPONSE * queryResponderUsingCurl(char *url, OCSP_CERTID *certid, c
   {
     if (!ACTIVATE_SSD)
     {
+      encodeUrlData(ocsp_req_base64, ocsp_req_base64_len, &encoded_ocsp_req_base64);
       snprintf(urlbuf, sizeof(urlbuf),
                ocsp_cache_server_retry_url_pattern,
-               host, ocsp_req_base64);
+               host, encoded_ocsp_req_base64);
```

Comment:
Checked entire sf_ocsp.c this seems to be the only place put base64 encoded ocsp request data into URL.